features/aragon/aragonFormError.tsx

```diff
@@ -0,0 +1,17 @@
+import { InputGroupStyled, InputNumber, Main } from 'shared/ui';
+import { Button } from '@lidofinance/lido-ui';
+import { FC } from 'react';
+
+export const AragonFormError: FC<{ text?: string }> = ({ text }) => {
+  return (
+    <Main.Card>
+      <InputGroupStyled fullwidth>
+        <InputNumber fullwidth label="Vote ID" disabled />
+      </InputGroupStyled>
+
+      <Button fullwidth type="submit" disabled={true}>
+        {text ?? 'Error'}
```

Comment:
use `text = 'Error'` in arguments instead

Reply:
Discussed in DM that these variants are not the same from code perspective, also would prefer to keep existing one since it's more common in TRP codebase